content/hi/cloud-computing.md

```diff
@@ -0,0 +1,24 @@
+---
+title: क्लाउड कंप्यूटिंग (Cloud Computing)
+status: Completed
+category: अवधारणा
+---
+
+## यह क्या है
+
+क्लाउड कंप्यूटिंग एक ऐसा मॉडल है जो इंटरनेट पर आवश्यकता के अनुसार सीपीयू, नेटवर्क और डिस्क क्षमताओं जैसे कंप्यूट संसाधनों की पेशकश करता है।
+और अन्य सभी तृतीय पक्षों को कई भौगोलिक स्थानों में संसाधनों की गणना करने के लिए किराए पर लेने की क्षमता प्रदान करते हैं।
```

Comment:
Hi @VaibhavJangid please check https://glossary.cncf.io/cloud-computing/ 
There is updated content for the cloud-computing page in english. We would like to capture that in your PR.